DSCPullServerAdmin/src/CmdLets/RemoveDSCPullServerAdminRegistration.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Isam.Esent.Interop;
+using DSCPullServerAdmin.src.Models;
+using DSCPullServerAdmin.src.Helpers;
+
+namespace DSCPullServerAdmin.src.CmdLets
+{
+    [Cmdlet(VerbsCommon.Remove,
+        "DSCPullServerAdminRegistration")]
```

Comment:
could you implement SupportsShouldProcess? I'd like to have WhatIf available for cmdlets that change data.
Also ConfirmImpact would be high since this is a destructive operation